aruco_opencv/package.xml

```diff
@@ -36,6 +36,7 @@
   <exec_depend>tf2_ros</exec_depend>
   <exec_depend>tf2_geometry_msgs</exec_depend>
   <exec_depend>yaml-cpp</exec_depend>
+  <exec_depend>compressed_image_transport</exec_depend>
```

Comment:
We don't really need this dependency here. It's only needed for publishing compressed images using image_transport.